db/schema.rb

```diff
@@ -57,6 +57,7 @@
   add_index "orders", ["user_id"], name: "index_orders_on_user_id", using: :btree
 
   create_table "users", force: :cascade do |t|
+    t.string   "name"
```

Comment:
User already has a name with the user_profile table. Any reason it belongs here instead / in addition?